Changes that correspond with "v5" commit of "chargio" branch in ManageIQ repo

gtanzillo opened the pull request:
See https://github.com/ManageIQ/manageiq/pull/15983

gtanzillo committed:
Changes that correspond with "v5" commit of "charge" branch in ManageIQ repo

lpichler committed:
add coverage for calculate_list_of_costs_input

lpichler committed:
method calculate_total_cost_input is replaced by calculate_list_of_costs_input

lpichler commented:
@chargio please review, added  coverage for calculate_list_of_costs_input and I remove the old one

ghost commented:
Hi @sergio-ocon, it seems like somebody is looking for you regarding chargeback!

lpichler committed:
Add missing methods for calculating metric values

added only for fixing spec, functionality is not provided
for gathering values from metric rollups

lpichler committed:
Add missing unit for metrics for old chargeback integration

lpichler committed:
Fix correct count of metrics

lpichler commented:
@gtanzillo @sergio-ocon 
This PR is adding method `calculate_list_of_costs_input` and adding missing metrics from old chargeback to map them on new chargeback.

PR can be unwiped (I dont'have permission).

lpichler committed:
Merge branch 'master' into chargio

chargio committed:
Merge branch 'master' into chargio
